ledger-core/virtual/integration/method_test.go

```diff
@@ -1993,3 +1993,236 @@ func TestVirtual_Method_IntolerableCallChangeState(t *testing.T) {
 	server.Stop()
 	mc.Finish()
 }
+
+// Send VStateReport with Dirty == Validated state
```

Comment:
Let think about using https://github.com/onsi/ginkgo to describe steps and checks

Reply:
I think we must discuss it on core meeting and create a task for refactoring.